services/ui-src/src/containers/Amendments.js

```diff
@@ -1,5 +1,6 @@
 import React, { useRef, useState, useEffect } from "react";
 import { useParams, useHistory } from "react-router-dom";
+import axios from "axios";
```

Comment:
im sure that this (axios) is available in the node modules because some other package is using it. But we should either install it explicitly, or just use the fetch api.

Reply:
ok I'll use fetch as oppose to axios